pbspro.spec

```diff
@@ -513,7 +513,6 @@ fi
 %exclude %{pbs_prefix}/lib*/python/pbs_bootcheck*
 %exclude %{pbs_prefix}/libexec/install_db
 %exclude %{pbs_prefix}/libexec/pbs_habitat
-%exclude %{pbs_prefix}/libexec/pbs_schema_upgrade
```

Comment:
This change is not related to this PR. I think someone else has deleted the line but forgot to update the pbspro.spec file.

Reply:
The same change is present in #905. Depending on which PR gets merged first, the other will already have the change in place.